PypeS/pypepad.py

```diff
@@ -465,16 +454,14 @@ def BuildMainFrame(self):
         self.output_to_file = StringVar()
         self.output_to_file.set('n')
  
-        scriptmenu = Menu(menu,tearoff=0,bd=1,activeborderwidth=0)
-        modulenames = ['vmtkscripts']
+        modulenames = ['vmtk.vmtkscripts']
+        splitIdentifiers = ['branch', 'centerline', 'image', 'surface', 'mesh']
```

Comment:
have a fallback case where we can collect all scripts that do not conform to the split scheme. Or, add a field in the script that explicitly encodes the category (which might be a lot better).